qpp-data-restoration/pom.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>gov.cms.qpp.conversion</groupId>
+	<artifactId>qpp-data-restoration</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
```

Comment:
can you make sure the version matches the version in the other POM files?

Reply:
Sure can.